scripts/tests/test_scripts.py

```diff
@@ -298,3 +298,32 @@ def test_pretrain_hvd():
         time.sleep(5)
     except ImportError:
         print("The test expects master branch of MXNet and Horovod. Skipped now.")
+
+@pytest.mark.serial
+@pytest.mark.gpu
+@pytest.mark.remote_required
+@pytest.mark.integration
+def test_finetune():
+    # WNLI training
+    process = subprocess.check_call(['python', './scripts/bert/finetune_classifier.py',
+                                     '--task_name', 'WNLI',
```

Comment:
parameterize the task name

Reply:
does WNLI not need the length argument?

Reply:
length is optional. I didn't parameterize task name because I don't want to actually train on MNLI/STS-B which takes quite a while.

Reply:
You can make whether to add inference_only as an argument to the test function too. See some of the other refactored tests for example

Reply:
I don't think testing all combinations of parameters for scripts bring extra value here (at the cost of making integration test longer)... If you insist, I have no problem adding them

Reply:
It shouldn't have to run all combinations as you can use parameterize to test only the combinations that you think are helpful. My comments are mostly from the perspective of writing general and concise test cases.

Reply:
I had some misunderstanding. Will update it